utopia-remix/app/services/fgaService.server.ts

```diff
@@ -0,0 +1,109 @@
+import { CredentialsMethod, OpenFgaClient } from '@openfga/sdk'
+import { ServerEnvironment } from '../env.server'
+import { AccessLevel } from '../types'
+
+export const fgaClient = new OpenFgaClient({
+  apiScheme: 'https',
+  apiHost: 'api.us1.fga.dev',
+  storeId: ServerEnvironment.FGA_STORE_ID,
+  // authorizationModelId: 'YOUR_MODEL_ID', // Optionally, you can specify a model id to target, which can improve latency
+  credentials: {
+    method: CredentialsMethod.ClientCredentials,
+    config: {
+      apiTokenIssuer: 'fga.us.auth0.com',
```

Comment:
honest question because I don't know: should this (and below) be configurable/put in the enviornment vars?

Reply:
added to env vars, just to prepare for future changes (currently those are the only values, but they can definitely change)